tools/virtualenvs/azure_open_envs-ansible2.9-python3.6-2022-01-10.txt

```diff
@@ -183,3 +183,6 @@ wrapt==1.12.1
 xmltodict==0.12.0
 yamllint==1.26.3
 zipp==3.5.0
+azure-identity==1.8.0
+msal==1.17.0
+msal-extensions==0.3.1
```

Comment:
as discussed in chat, we usually don't update py venv

the process is:
- create a new one, with a new name / date
- update readme.adoc  known issues   to describe the problem and the solution (move to the new pyvenv) for ex

Reply:
moved to new file